x-pack/plugins/snapshot_restore/server/routes/api/snapshots.ts

```diff
@@ -81,6 +168,8 @@ export function registerSnapshotsRoutes({
             repositories,
             // @ts-expect-error @elastic/elasticsearch "failures" is a new field in the response
             errors: fetchedSnapshots?.failures,
+            // @ts-expect-error @elastic/elasticsearch "total" is a new field in the response
```

Comment:
Could you please file an issue in https://github.com/elastic/elasticsearch-specification and add the link to the issue here?